tests/sqlparser_clickhouse.rs

```diff
@@ -1091,6 +1091,38 @@ fn parse_create_table_on_commit_and_as_query() {
     }
 }
 
+#[test]
+fn parse_select_table_function_settings() {
+    let sql = r#"SELECT * FROM table_function(arg, SETTINGS setting = 3)"#;
```

Comment:
Can we extend the test case to cover more than one setting?

Reply:
I added a setting. I didn't expand on the test too much, because the settings parsing is already more exhaustively tested by the tests for `SETTINGS` in queries